### PR TITLE
Refactor/bot responses graphql

### PR DESCRIPTION
**Pull request by GuillaumeTech**
just created to run the tests on the CI

- [ ] I wrote tests for the feature
- [ ] I documented the feature if needed

If the feature is a refactor, please explain why your way is better

### CODE REVIEW

File: botfront/imports/api/graphql/botResponses/schemas/botResponses.types.graphql [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+}
+
+type BotResponse {
+    key: String
+    _id: String
+    projectId: String
+    values: [Value]
+}
+
+type Value {
```
Comment (pheel) [inline botfront/imports/api/graphql/botResponses/schemas/botResponses.types.graphql]: I don't think we should name types with such a generic name.

File: botfront/imports/api/graphql/botResponses/schemas/botResponses.types.graphql [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+input ValueInput {
+    lang: String
+    sequence: [ContentContainerInput]
+}
+
+input ContentContainerInput {
+    content: String
+}
+
+input FollowUpInput {
```
Comment (pheel) [inline botfront/imports/api/graphql/botResponses/schemas/botResponses.types.graphql]: this one is deprecated

File: botfront/imports/api/graphql/botResponses/schemas/botResponses.types.graphql (hunk 1 above)
Comment (GuillaumeTech) [reply botfront/imports/api/graphql/botResponses/schemas/botResponses.types.graphql]: done

File: botfront/imports/api/graphql/botResponses/schemas/botResponses.types.graphql (hunk 2 above)
Comment (GuillaumeTech) [reply botfront/imports/api/graphql/botResponses/schemas/botResponses.types.graphql]: done

**Pull request merged**